wiz-broker/templates/secrets.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.global.broker.enabled }}
+{{- if and .Values.global.wizConnector.createSecret .Values.global.broker.createSecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "wiz-broker.connectorSecretName" . | trim }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "wiz-broker.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
```

Comment:
no need pre install here, use .Values.global.wizConnector.annotations for add it from k8s connector